doc/Standaard-inrichting-GitHub-GitLab.md

```diff
@@ -181,5 +198,5 @@ Meer informatie over het activeren van GitHub Pages specifiek voor VNG Realisati
 
 ### Spectral rules???
 
-### GitHub Pages???
+### GitLab Pages???
```

Comment:
Ik vraag me af of we bij de initiële inrichting van een repo wel al voorzieningen moeten inrichten voor een GitHub Pages omgeving. Meestal ontstaat de behoefte aan een GitHub Pages omgeving pas op een later tijdstip. Wellicht is het voldoende om pas op dat moment daarvoor de noodzakelijke voorzieningen in de repo aan te brengen. Wellicht beter om het aanbrengen van de noodzakelijke en optionele voorzieningen in het document [GitHub-Pages-implementatie-VNG-R.md](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/GitHub%20Pages/GitHub-Pages-implementatie-VNG-R.md) te beschrijven.